ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1034,6 +1035,21 @@ def test_source_ipahealthcheck_ds_backends(self, rename_ldif):
             assert check["result"] == "CRITICAL"
             assert exception_msg in check["kw"]["exception"]
 
+    def test_source_pki_server_clones_connectivity_and_data(self):
+        """
+        This testcase checks that when ClonesConnectivyAndDataCheck
+        is run it doesn't display source not found error
+        """
+        error_msg = (
+            "Source pki.server.healthcheck.clones.connectivity_and_data"
+            "not found"
```

Comment:
According to the BZ https://bugzilla.redhat.com/show_bug.cgi?id=2041995 the exact message contains simple quotes around the source name:
`Source 'pki.server.healthcheck.clones.connectivity_and_data' not found`

Reply:
The msg should be
```
         error_msg = ( 
             "Source 'pki.server.healthcheck.clones.connectivity_and_data' "
             "not found"
        )
```